compiler/src/dotty/tools/repl/ReplCompiler.scala

```diff
@@ -26,39 +27,43 @@ import scala.collection.mutable
 /** This subclass of `Compiler` replaces the appropriate phases in order to
  *  facilitate the REPL
  *
- *  Specifically it replaces the front end with `REPLFrontEnd`, and adds a
- *  custom subclass of `GenBCode`. The custom `GenBCode`, `REPLGenBCode`, works
- *  in conjunction with a specialized class loader in order to load virtual
- *  classfiles.
+ *  Specifically it replaces the front end with `REPLFrontEnd`.
  */
```

Comment:
I would just delete this whole comment, it doesn't really say anything interesting. (it could be replaced by an actual description of what REPLCompiler does though :))